lib/tes.js

```diff
@@ -22,9 +24,13 @@ class TES {
 
         TES._instance = this;
 
-        config.identity = config.identity ? config.identity : {};
-        config.listener = config.listener ? config.listener : {};
-        config.options = config.options ? config.options : {};
+        config.identity = config.identity || {};
+        config.listener = config.listener || {};
+        config.options = config.options || {};
+
+        config.path = config.path || defaultPath; // default path
```

Comment:
since `path` is related to the webhooks, it should probably be put inside the `config.listener` object